android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1926,15 +2060,36 @@ else if ( maximumAdContentRating == 3 )
         return AppLovinAdContentRating.NONE;
     }
 
-    private List<String> getStringArrayList(@Nullable ReadableArray readableArray)
+    private static int getRawAppLovinAdContentRating(final AppLovinAdContentRating maximumAdContentRating)
+    {
+        switch ( maximumAdContentRating )
+        {
+            case NONE:
+                return 0;
+            case ALL_AUDIENCES:
+                return 1;
+            case EVERYONE_OVER_TWELVE:
+                return 2;
+            case MATURE_AUDIENCES:
+                return 3;
+            default:
+                return 0;
+        }
+    }
+
+    @Nullable
+    private List<String> getStringArrayList(@Nullable final ReadableArray readableArray)
     {
         if ( readableArray == null ) return null;
 
         List<String> list = new ArrayList<>( readableArray.size() );
 
-        for ( Object item : readableArray.toArrayList() )
+        for ( int i = 0; i < readableArray.size(); i++ )
         {
-            list.add( (String) item );
+            if ( readableArray.getType(i) == String )
```

Comment:
`getType()` returns an enum of type `ReadableType`. `String` is a Java class so I'm surprised this compiles. It should probably be:

```
readableArray.getType( i ) == ReadableType.String
```

Also, formating.

Reply:
Thanks, I don't know what I'm thinking...